README.md

```diff
@@ -147,13 +150,13 @@ Everyone at Rapid River is invited to join the [Rapid River Slack](https://rapid
   * `#random`: Random, fun, and goofy stuff. Cat videos, bizarre news headlines, happenings from your town that might be of interest to other Rapid Riverers
   * `#devops`: Things of interest to practitioners of the dark DevOps magic.
   * `#farside`: The [best comic strip](https://www.thefarside.com/) ever.
-* Use [Slack threading](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) to stay on topic. This is a major thing! It's a lot easier to read a channel when discussions get moved into threads for follow-up conversation.
-* Emojis can be an effective way to express approval, surprise, joy, etc. instead of messages. Like threads, that reduces clutter in the channel and keeps communication flowing
+* Use [Slack threading](https://slack.com/help/articles/115000769927-Use-threads-to-organize-discussions-) to stay on topic. This is a major thing! It's a lot easier to read a channel when discussions get moved into threads for a follow-up conversation.
```

Comment:
1. `a follow-up conversation` is what I intended.
2. `keeps communication flowing` is the correct grammatical match for the subject of the sentence, `that`. The better edit would be to replace `that reduces` with `they reduce` in which case `keep communication flowing` would become correct.

Reply:
Please check again. 

The original sentence is grammatically correct:

> Like threads, that reduces clutter in the channel and keeps communication flowing.

I suggested an alternative that would be grammatically correct and sounds nicer:

> Like threads, they reduce clutter in the channel and keep communication flowing.

But your suggestion is a mix of these two and is grammatically incorrect (`that reduces` (singular subject) has a  mismatch with `keep` (plural conjugation) of `to keep`).

> Like threads, that reduces clutter in the channel and keep communication flowing.